spec/models/devotional_spec.rb

```diff
@@ -49,4 +49,12 @@
       )
     end
   end
+
+  describe '.get_devotionals' do
+    it 'returns a list of devotionals' do
+      create_list(:devotional, 6)
+
+      expect(described_class.get_devotionals.count).to eq(6)
```

Comment:
Devotional.size would be a little better, that way you don't need to change 6 twice if you ever wanted to update this test.

Reply:
On the right side? (the to eq part?) Says Devotional does not have method "size". I did just change the left side to described_class.count though

Reply:
Ok I'm not crazy, it's a thing.
https://medium.com/unagi/ruby-on-rails-length-vs-size-vs-count-with-examples-a32e17274c79
It might be Rails-specific for some reason but there could be performance implications.

To be honest though, this test is really testing the framework. We are very confident that when you do Devotional.all it will return all of them, so we don't need to test this explicitly here now that we don't have that class method.
The fix here is just to delete the model test. What we want to focus testing on is the API response.

Reply:
Gotcha! 🫡